extension/oauth2clientauthextension/README.md

```diff
@@ -74,7 +74,11 @@ Following are the configuration fields
 
 - [**token_url**](https://datatracker.ietf.org/doc/html/rfc6749#section-3.2) - The resource server's token endpoint URLs.
 - [**client_id**](https://datatracker.ietf.org/doc/html/rfc6749#section-2.2) - The client identifier issued to the client.
+- **client_id_file** - The file path to retrieve the client identifier issued to the client.
+  This setting takes precedence over `client_id`.
```

Comment:
It would be nice to clarify how (and whether) the values are refreshed when the backing file has changed.

Reply:
Ack